src/instructions/auf_dem_trockenen.md

```diff
@@ -12,7 +12,7 @@
 
 Welcome to the Dry Land Use Case.
 
-In this example, the GBIF occurrence data of *Calopteryx splendens* are clipped to the extent of Germany and merged with the land use classification from the Oekosystematlas as well as a time of average temperature provided by the WorldClim dataset.
+In this example, the GBIF occurrence data of *Calopteryx splendens* are clipped to the extent of Germany and merged with the land use classification from the IÖR land use classification as well as aggregating daily resolved average temperature data provided by the German weather service (DWD).
```

Comment:
I think I would remove the "daily resolved average" here and just write "... as well as aggregated temperature data ...". The way I read it now it sort of implies that you perform the aggregation to a daily average. Later when you actually perform an aggregation you can say that the data already comes as a daily aggregation and you further aggregate it. I think this would make it clearer.